pkg/cmd/list.go

```diff
@@ -59,85 +67,107 @@ func NewCmdWhoCan() *cobra.Command {
   kubectl who-can create services -n foo
 
   # List who can get the service named "mongodb" in namespace "bar"
-  kubectl who-can get svc mongodb --namespace bar`,
+  kubectl who-can get svc mongodb --namespace bar
+
+  # List who can read pod logs
+  kubectl who-can get pods --subresource=log`,
 		Args: func(cmd *cobra.Command, args []string) error {
 			if len(args) < 2 {
 				return errors.New("please specify at least a verb and a resource type")
 			}
 			return nil
 		},
-		Run: func(cmd *cobra.Command, args []string) {
-			namespace, _, err := configFlags.ToRawKubeConfigLoader().Namespace()
-			if err != nil {
-				fmt.Printf("Error getting namespace from context config: %v\n", err)
-				os.Exit(1)
+		SilenceUsage: true,
```

Comment:
Is it deliberate to silence usage here?

Reply:
Yup. I wanted to be consistent with the built-in kubectl commands and existing plugins:

```
$ kubectl auth can-i go home
Warning: the server doesn't have a resource type 'home'
yes
$ kubectl get swimming-pools
error: the server doesn't have a resource type "swimming-pools"
$ ./kubectl-who-can go home
Error: resolving resource: the server doesn't have a resource type "home"
```